### ISSUE TITLE
Frontend Login doesn't show error messages

### ISSUE THREAD

**Issue by christian-kolb** [contao/core-bundle#610]
I've got a Contao installation running with Symfony 3.1.6 and Contao 4.2.5.

When using the frontend login it's possible to login as normal. But when using the wrong username password combination it doesn't show any kind of error message. It just shows the login page again.

I looked into the code and found the following error check:

```
if (isset($_SESSION['MESSAGES'][TL_MODE]['TL_ERROR']))
{
    $blnHasError = true;
    $_SESSION['LOGIN_ERROR'] = $_SESSION['MESSAGES'][TL_MODE]['TL_ERROR'][0];
    unset($_SESSION['MESSAGES'][TL_MODE]['TL_ERROR']);
}

if (isset($_SESSION['LOGIN_ERROR']))
{
    $blnHasError = true;
    $this->Template->message = $_SESSION['LOGIN_ERROR'];
}
```

When I print the `$_SESSION` variable the output look like this:

```
Array
(
    [_sf2_attributes] => Array
        (
            [_locale] => de
            [_csrf/contao_csrf_token] => ZzrSu23hDQF5mkDfuO3zbzwI8wsmErXxCEEBhDy2qoQ
        )

    [_sf2_flashes] => Array
        (
            [contao.FE.error] => Array
                (
                    [0] => Anmeldung fehlgeschlagen (Benutzernamen und Passwörter unterscheiden zwischen Groß- und Kleinschreibung)!
                )
        )
    [_contao_be_attributes] => Array
        (
        )
    [_contao_fe_attributes] => Array
        (
        )
    [_sf2_meta] => Array
        (
            [u] => 1478011003
            [c] => 1478010749
            [l] => 0
        )
    [BE_DATA] => Contao\CoreBundle\Session\Attribute\ArrayAttributeBag Object
        (
            [name:Symfony\Component\HttpFoundation\Session\Attribute\AttributeBag:private] => contao_backend
            [storageKey:Symfony\Component\HttpFoundation\Session\Attribute\AttributeBag:private] => _contao_be_attributes
            [attributes:protected] => Array
                (
                )
        )
    [FE_DATA] => Contao\CoreBundle\Session\Attribute\ArrayAttributeBag Object
        (
            [name:Symfony\Component\HttpFoundation\Session\Attribute\AttributeBag:private] => contao_frontend
            [storageKey:Symfony\Component\HttpFoundation\Session\Attribute\AttributeBag:private] => _contao_fe_attributes
            [attributes:protected] => Array
                (
                )
        )
    [TL_LANGUAGE] => de
)
```

Is it possible that there was a change in one of the components that other keys or another structure is used there? Is this a "simple bug" or is there something that can be configured for it to work?

**Comment by christian-kolb** [comment]
Thanks for adding the issue to the list. Why is it "only" the next feature version? Isn't that a simple bug that would be fixable with a bugfix release?

**Comment by leofeyer** [comment]
Sure, but I doubt that there will be another bugfix release for Contao 4.2. There is already a release candidate of Contao 4.3 and we are expecting the final release in less than 4 weeks.

**Comment by christian-kolb** [comment]
Ok, yes that makes sense. So this will be fixed within this 4 week timeframe?

**Comment by leofeyer** [comment]
Fixed in f8c5e556303fcec7aba228ad05b6ecea0b18595c.
